zinc/src/main/scala/sbt/inc/IncrementalCompilerUtil.scala

```diff
@@ -1,17 +0,0 @@
-/*
- * Zinc - The incremental compiler for Scala.
- * Copyright 2011 - 2017, Lightbend, Inc.
- * Copyright 2008 - 2010, Mark Harrah
- * This software is released under the terms written in LICENSE.
- */
-
-package sbt
-package inc
-
-import xsbti.compile.IncrementalCompiler
-import sbt.internal.inc.IncrementalCompilerImpl
-
-object IncrementalCompilerUtil {
```

Comment:
This was moved because we need an api of utils from which people can get default implementations without exposing the Scala implementations. This move is the first step towards this direction.